core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2020,6 +2020,14 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  private[spark] val AUTO_STOP_ACTIVE_SPARK_CONTEXTS =
+    ConfigBuilder("spark.kubernetes.submit.autoStopActiveSparkContexts")
+      .version("3.4.0")
```

Comment:
`3.5.0`? Also, cc @xinrong-meng because she is the release manager of Apache Spark 3.4.0.